### PR TITLE
Gets rid of the keyboard when you do search

### PR DESCRIPTION
**Pull request by WiegnerO**

### CODE REVIEW

File: app/src/main/java/com/straats/ben/friendrequest/Utils.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
     //Not sure if these constants should be in Utils or VolleyWrapper
     //URLs Section
-    protected final static String baseURL = "http://api.friendrequest.ca/";
+    protected final static String baseURL = "http://192.168.2.25:3030/";
```
Comment (benstraats) [inline app/src/main/java/com/straats/ben/friendrequest/Utils.java]: revert this change

File: app/src/main/java/com/straats/ben/friendrequest/SearchListHelper.java [hunk 2]
```diff
@@ -166,6 +193,7 @@ public void onClick(View v) {
             }
         });
 
+
```
Comment (benstraats) [inline app/src/main/java/com/straats/ben/friendrequest/SearchListHelper.java]: No need for this extra line here

**Comment by WiegnerO** [comment]
test comment

**Review by benstraats — CHANGES_REQUESTED** [review]

**Review by benstraats — APPROVED** [review]
